script/Ciel/Day 9/09_01_CIEL09_4.txt

```diff
@@ -640,6 +641,7 @@ Or rather, that means―――
 [sha:0af644c9c20811b245f9baf1b4d04b3070666958]{
 -- Page 417, Offset 23210.
 -- 「年齢が合わない、でしょ？
+// How to handle the fact that nenrei is just ciel
```

Comment:
maybe something like:
> You're wondering why her age doesn't add up, right?
> You're thinking her age doesn't line up, right?

or more literally:
> You're thinking her age doesn't fit, right?

Reply:
I like the first one